docs/general/ddl/sharding.rst

```diff
@@ -62,11 +62,18 @@ is applied, (see :ref:`ref_clustered_clause`).
 Routing
 =======
 
-CrateDB algorithmically determines which shard each table row belongs to. It
-does this using the value of a routing column.
+Given a fixed number of primary shards, individual rows can be routed to a
+fixed shard number using a simple formula:
 
-The column used for routing can be freely defined using the ``CLUSTERED BY
-(<column>)`` statement and is used to route a row to a particular shard.
+*shard number = hash(routing column) % total primary shards*
+
+When hash values are distributed evenly (which will be approximately true in
+most cases) rows will be distributed evenly amongst the fixed amount of
+available shards.
+
+The routing column can be specified defined with the ``CLUSTERED BY
```

Comment:
```suggestion
The routing column can be defined with the ``CLUSTERED BY
```

Reply:
I don't think "defined" is correct here

here's my understanding of the two words in a technical context:

- "define" - describe a specific noun (i.e., describe what it is)
- "specify" - highlight/choose a specific noun for some purpose (i.e., something that has been defined elsewhere)

in this case, the column is *defined* in the table definition. its definition is `first_column integer` (for example). the `CLUSTERED BY` clause *specifies* that the (pre-defined) `first_column` column is to be used for shard routing

Reply:
Sure. Then delete `defined`.

```suggestion
The routing column can be specified with the ``CLUSTERED BY
```

Reply:
omg I totally missed that 😅 thanks!